frontend/src/metabase-lib/lib/queries/structured/MBQLClause.js

```diff
@@ -115,6 +115,14 @@ export class MBQLObjectClause {
   metadata() {
     return this._query.metadata();
   }
+
+  raw() {
+    const entries = Object.entries(this).filter(entry => {
```

Comment:
Might be helpful to call this const something more specific, especially as we use `.entries` and later `.fromEntries`.

Just a silly idea: `entriesWithDefinedValue`?

Reply:
SGTM, thanks 🤝